uitools/toolkitcpp/README.md

```diff
@@ -0,0 +1,192 @@
+[![ArcGIS Maps SDK for Qt Toolkit API reference](https://img.shields.io/badge/API_Reference-purple)](https://developers.arcgis.com/qt/latest/toolkit/api-reference/) [![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt)
+
+## ArcGIS Maps SDK - Qt Quick UI components
+
+These are controls that compliment your `MapView` or `SceneView` by adding them to your Qt project's .qml files (QML based UI development). They are components like: AuthenticationView, NorthArrow, OverviewMap, TimeSlider, and so forth. Using these components are very beneficial for device based apps like phones and tablets.
+
+You can learn more about the benefits of building apps with the [Qt Quick UI controls](https://doc.qt.io/qt-6/qtquickcontrols-index.html) from the Qt Group. 
+
+## Available components
+
+These are the components available to use:
+
+- **[AuthenticationView - Qt Quick UI control/QML Type](docs/AuthenticationView.md)** - A wrapper view that will automatically display the proper authentication view for any of the supported authentication types (OAuth, Token, HTTP Basic, HTTP Digest, SAML, PKI).
```

Comment:
I see the terms "Qt Quick UI control/QML Type" are repeated with each component's hyperlink. 
Could this be simplified to instead appear only once in the lead-in description:

```suggestion
The following Qt Quick UI control/QML Type components are available to use:

- **[AuthenticationView](docs/AuthenticationView.md)** - A wrapper view that will automatically display the proper authentication view for any of the supported authentication types (OAuth, Token, HTTP Basic, HTTP Digest, SAML, PKI).
```

Reply:
Addressed with commit: https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/660/commits/8be83f47a4d3d108102e0b160a03de129edacf54